crates/turborepo-lib/src/package_changes_watcher.rs

```diff
@@ -171,9 +171,29 @@ impl Subscriber {
             return None;
         };
 
+        // Check for both turbo.json and turbo.jsonc
+        let turbo_json_path = self.repo_root.join_component(CONFIG_FILE);
+        let turbo_jsonc_path = self.repo_root.join_component(CONFIG_FILE_JSONC);
+
+        let turbo_json_exists = turbo_json_path.exists();
+        let turbo_jsonc_exists = turbo_jsonc_path.exists();
+
+        let config_path = match (turbo_json_exists, turbo_jsonc_exists) {
+            (true, true) => {
+                tracing::warn!(
+                    "Found both turbo.json and turbo.jsonc in {}. Using turbo.json for watching.",
+                    self.repo_root
+                );
+                turbo_json_path
+            }
+            (true, false) => turbo_json_path,
+            (false, true) => turbo_jsonc_path,
+            (false, false) => turbo_json_path, // Default to turbo.json if neither exists
+        };
```

Comment:
Not blocking, but would love to dedupe this logic with `Config::root_turbo_json_path` instead of copy-pasta. Would need to break out the behavior to a new static function.